app/mailers/contact_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class ContactMailer < ActionMailer::Base
+  include EmailHeaderHelper
+
+  helper ApplicationHelper
+
+  def subscription_notification(contact)
+    @contact = contact
+
+    subject = "You have been added to codebar's sponsors mailing list"
+
+    mail(mail_args(contact, subject, 'no-reply@codebar.io'), &:html)
+  end
+
+  private
+
+  helper do
```

Comment:
@despo the same method has been already defined in `EmailHelper`, should we perhaps use that here?

Reply:
👍

Reply:
👍